src/Post/ReadingTime.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+namespace P4\MasterTheme\Post;
+
+use IntlBreakIterator;
+use DOMDocument;
+use DOMXpath;
+
+/**
+ * This class calculates a reading time from a post content.
+ */
+class ReadingTime {
```

Comment:
If I see this class name I would expect an instance to be an actual reading time. Maybe `ReadingTimeCalculator` would be better here.

Another option is to make the content an argument of `ReadingTime` constructor. Since `wpm` and `locale` are already constructor arguments, this kind of makes sense.